cves/CVE-2019-6975.yml

```diff
@@ -218,37 +239,41 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
-    note:
+    applies: true
+    note: |
```

Comment:
Sound logic, does django itself use this anywhere else though?

Reply:
This was used by some other format functions, which I mentioned earlier as unit tests existed for those other functions. I was unable to find where else the offending function may be invoked.